client/src/components/Form/FormCard.vue

```diff
@@ -28,10 +27,6 @@ export default {
             type: String,
             required: false,
         },
```

Comment:
Add a default if your prop is not required

Reply:
Good catch, thanks.